README.md

```diff
@@ -43,32 +47,27 @@ for definition of maturity levels).
 
 Components marked `Stable` provide the following guarantees:
 
-- Field types will not change.
-- Field numbers will not change.
+- Field types, numbers and names will not change.
 - Numbers assigned to enum choices will not change.
 - Service names and service package names will not change.
 - Service operation names, parameter and return types will not change.
 
 The following changes are allowed:
 
 - Message names may change.
-- Field names may change.
 - Enum names may change.
 - Enum choice names may change.
```

Comment:
Can you clarify why here by adding a link to the specification.

Reply:
Sorry, I don't understand. What do you want to link to and from where?

Reply:
```suggestion
- Enum choice names may change. This is allowed because enum names are not sent on the wire.
```

Reply:
Done